R/visit.R

```diff
@@ -1,7 +1,7 @@
 #' Visits the set of nodes and all of their children, invoking the callback for each visited node
 #'
 #' @param nodes The list or array of nodes to visit
-#' @param callback The callback function to invoke for each node
+#' @param callback The callback function to invoke for each node. The callback should return TRUE to continue visiting the children of the node, or FALSE to stop visiting the children of the node.
```

Comment:
Maybe extend this to "anything non-false"? Or do we want a check to enforce `\(n) { magic(n); return TRUE }`

Reply:
Nah, I don't like requiring an explicit return to indicate default behavior. I fixed the doc comment.